mastering/version_ranges.rst

```diff
@@ -21,26 +21,25 @@ There are some packages that do not follow semver, a popular one would be the Op
 The process to manage plain versions vs version-ranges is also different. The second one requires a "search" in the remote, which is orders of magnitude slower than direct retrieval of the reference (plain versions), so take it into account if you plan to use it for very large projects.
 
 
-Expressions are those defined and implemented by https://pypi.org/project/node-semver/,
-but using a comma instead of spaces. Accepted expressions would be:
+Expressions are those defined and implemented by https://pypi.org/project/node-semver/. Accepted expressions would be:
 
 ..  code-block:: python
 
-   >1.1,<2.1    # In such range
-   2.8          # equivalent to =2.8
-   ~=3.0        # compatible, according to semver
-   >1.1 || 0.8  # conditions can be OR'ed
+   [>1.1 <2.1]    # In such range
```

Comment:
I think maybe it would be more clear for users to keep the square brackets

Reply:
I already added the square brackets!?

Reply:
Sorry, I misunderstood the diff